### PR TITLE
MoviePage CSS changed

### PR DESCRIPTION
**Pull request by GrzegorzPacewicz**

### CODE REVIEW

**Comment by GrzegorzPacewicz** [comment]
I tried to fix css on movie page. Please check it.

**Review by greedyChicken — APPROVED** [review]
ok, looks clean, good job

**Comment by GrzegorzPacewicz** [comment]
I've also resolved problem with bug I mentioned on meeting. It was on Jason Statham page.

**Review by greedyChicken — APPROVED** [review]
I think now it's better than ever

**Comment by GrzegorzPacewicz** [comment]
I've added few small changes in styles

File: src/core/theme.js [hunk 1]
```diff
@@ -19,9 +19,12 @@ export const theme = {
     medium: 767,
     large: 1024,
     extraLarge: 1199,
+    bigest: 1389,
```
Comment (bartekdbc) [inline src/core/theme.js:23]: It is good idea, but it should be "biggest" or "theBiggest" as it is in English.

**Review by bartekdbc — APPROVED** [review]
Good work with fixing css, our app looks much better now.

**Pull request merged**